PusherClient/EventEmitter.cs

```diff
@@ -115,39 +171,42 @@ public void UnbindAll()
 
             _rawEventListeners.Clear();
             _rawGeneralListeners.Clear();
+
+            _pusherEventEventListeners.Clear();
+            _pusherEventGeneralListeners.Clear();
         }
 
-        internal void EmitEvent(string eventName, string data)
+        internal void EmitEvent(string eventName, PusherEvent data)
         {
-            foreach (var action in _rawGeneralListeners)
-            {
-                action(eventName, data);
-            }
+            var stringData = data.ToString();
 
-            if (_rawEventListeners.ContainsKey(eventName))
+            dynamic thing = new Dictionary<string, object>

```

Comment:
I think this is some left over testing code :)

Reply:
Whoops! :) ty :)